operating/examine-images-for-vulnerabilities.adoc

```diff
@@ -7,17 +7,28 @@ include::modules/common-attributes.adoc[]
+With {product-title} you can analyze images for vulnerabilities using the default {product-title-short} scanner, or you can xref:../integration/integrate-with-image-vulnerability-scanners.adoc#integrate-with-image-vulnerability-scanners[configure an integration] to use another supported scanner. The scanners in {product-title-short} analyze each image layer to find packages which we then match against known vulnerabilities by comparing them with a vulnerability database populated from different sources. Depending on the scanner used, sources include the National Vulnerability Database (NVD), the Open Source Vulnerabilities (OSV) vulnerability database, and operating system vulnerability feeds.
 
-When Scanner finds any vulnerabilities, it:
+[NOTE]
+====
+The {product-title-short} Scanner V4 uses the OSV database available at link:https://osv.dev/[OSV.dev] under license.
+====
 
-* Shows them in the xref:../operating/manage-vulnerabilities/vulnerability-management.adoc#vulnerability-management[*Vulnerability Management*] view for detailed analysis.
-* Ranks vulnerabilities according to risk and highlights them in the RHACS portal for risk assessment.
-* Checks them against enabled xref:../operating/manage-security-policies.adoc#manage-security-policies[security policies].
+{product-title-short} contains two scanners: the StackRox Scanner and Scanner V4. The StackRox Scanner originates from a fork of the Clair v2 open source scanner and is the default scanner. In release 4.4, {product-title-short} introduced Scanner V4, built on ClairCore, which provides additional image scanning features.
 
-Scanner inspects the images and identifies the installed components based on the files in the images.
-It may fail to identify installed components or vulnerabilities if the final images are modified to remove the following files:
+[NOTE]
+====
+This documentation uses the term "{product-title-short} scanner" or "Scanner" to refer to the combined scanning capabilities provided by the two scanners: the StackRox Scanner and Scanner V4. When referring to the capabilities of a specific scanner, the name of the specific scanner is used.
+====
+
+When the {product-title-short} scanner finds any vulnerabilities, it performs the following actions:
+
+* Shows them in the xref:../operating/manage-vulnerabilities/vulnerability-management.adoc#vulnerability-management[*Vulnerability Management*] view for detailed analysis
+* Ranks vulnerabilities according to risk and highlights them in the {product-title-short} portal for risk assessment
+* Checks them against enabled xref:../operating/manage-security-policies.adoc#manage-security-policies[security policies]
+
+The {product-title-short} scanner inspects the images and identifies the installed components based on the files in the images.
+It might fail to identify installed components or vulnerabilities if the final images are modified to remove the following files:
 
 [cols=",a"options="header"]
 |===
```

Comment:
this list is a bit outdated, but I understand it's cutting it very close. Can you make a Jira to update this?

Reply:
https://issues.redhat.com/browse/ROX-23212